src/test/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/AwsAccountingXmlDocumentAccountingLineServiceImplTest.java

```diff
@@ -0,0 +1,133 @@
+package edu.cornell.kfs.fp.batch.xml.cloudcheckr;
+
+import edu.cornell.kfs.fp.batch.service.AwsAccountingXmlDocumentAccountingLineService;
+import edu.cornell.kfs.fp.batch.xml.AccountingXmlDocumentAccountingLine;
+import edu.cornell.kfs.fp.batch.xml.DefaultKfsAccountForAws;
+import edu.cornell.kfs.fp.batch.xml.cloudcheckr.fixture.GroupLevelFixture;
+import edu.cornell.kfs.fp.batch.xml.fixture.AccountingXmlDocumentAccountingLineFixture;
+import edu.cornell.kfs.fp.batch.xml.fixture.DefaultKfsAccountForAwsFixture;
+
+import org.apache.commons.lang.ObjectUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.kuali.kfs.sys.ConfigureContext;
+import org.kuali.kfs.sys.context.KualiTestBase;
+import org.kuali.kfs.sys.context.SpringContext;
+
+@ConfigureContext(session = org.kuali.kfs.sys.fixture.UserNameFixture.ccs1)
```

Comment:
I'd suggest reworking the unit tests so Spring isn't required.  We can mock the various services that AwsAccountingXmlDocumentAccountingLineService uses.  There are a couple of ways to do this, we can peer code this if you'd like.